src/backend/src/services/users.services.ts

```diff
@@ -52,9 +53,15 @@ export default class UsersService {
       create: { userId }
     });
 
+    const secureSettings = await prisma.user_Secure_Settings.upsert({
+      where: { userId },
+      update: {},
+      create: { userId, nuid: `${userId}`, street: '', city: '', state: '', zipcode: '', phoneNumber: `${userId}` }
```

Comment:
this is not how we should be doing this, you should update the schema to make the unique fields optional, and just add default values for street, city, state, and zipcode

Reply:
r u sure?

Reply:
I think what we should do instead is just make a fake secureSettings object and then not touch the database if yk what i mean

Reply:
Which route would you guys prefer I go?